client/agenda/store.js

```diff
@@ -154,6 +154,29 @@ export const useAgendaStore = defineStore('agenda', {
           this.timezone = agendaData.meeting.timezone
         }
 
+        // The following is for developers to modify `agendaData` dates to match
+        // your current system time so that features depending on an overlap of times
+        // such as the 'Now' button and red divider are available.
+        // This should be commented out when not in 
+        // (function(){
```

Comment:
This is the developer-only utility to modify server agenda dates to be the same as local browser time, so that the 'Now' button is available.

I could make it a utility function and add tests etc, or make it active with a specific url param or something like that, but I'm not sure how far to go with it. What approach does this codebase usually take?

Reply:
There's not a pattern to follow.

Reply:
You _could_ follow the python settings.SERVER_MODE, or have a separate front-end server mode setting but I don't the the LOE required is the right investment right now. The danger of having a commented out block like this is that it will rot without CI noticing.

Reply:
You can already modify the current time from the agenda settings panel for debugging:

![image](https://github.com/user-attachments/assets/ebbcc8d9-565f-43ae-b7e5-45269caa4435)

Reply:
This block has been removed in favour of the debugging panel